docs/source/conf.py

```diff
@@ -29,6 +31,17 @@
     "hoverxref.extension",
 ]
 
+nbsphinx_execute = "auto"  # auto, never
+
+highlight_language = "python3"
+
+nbsphinx_execute_arguments = [
+    "--InlineBackend.figure_formats={'png2x'}",
+    "--InlineBackend.rc=figure.dpi=96",
+]
+
+nbsphinx_kernel_name = "python3"
+
```

Comment:
This is not required since no jupyter notebooks are used in the documentation.

Reply:
done 👍🏻